tests/phpunit/tests/rest-api/rest-plugins-controller.php

```diff
@@ -0,0 +1,1108 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Plugins_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST API
+ */
+
+/**
+ * @group restapi
+ */
+class WP_REST_Plugins_Controller_Test extends WP_Test_REST_Controller_Testcase {
+
+	const BASE        = '/wp/v2/plugins';
+	const PLUGIN      = 'test-plugin/test-plugin';
+	const PLUGIN_FILE = self::PLUGIN . '.php';
+
+	/**
+	 * Subscriber user ID.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @var int
+	 */
+	private static $subscriber_id;
+
+	/**
+	 * Super administrator user ID.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @var int
+	 */
+	private static $super_admin;
+
+	/**
+	 * Administrator user id.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @var int
+	 */
+	private static $admin;
+
+	/**
+	 * Set up class test fixtures.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param WP_UnitTest_Factory $factory WordPress unit test factory.
+	 */
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$subscriber_id = $factory->user->create(
+			array(
+				'role' => 'subscriber',
+			)
+		);
+		self::$super_admin   = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		self::$admin         = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+
+		if ( is_multisite() ) {
+			grant_super_admin( self::$super_admin );
+		}
+	}
+
+	/**
+	 * Clean up test fixtures.
+	 *
+	 * @since 5.5.0
+	 */
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$subscriber_id );
+		self::delete_user( self::$super_admin );
```

Comment:
Looks to be missing a `delete_user` for `self::$admin`?

Reply:
Fixed in 385b48c.